app/javascript/components/widgets/numbers/falling-blocks.jsx

```diff
@@ -143,6 +182,15 @@ class Scene extends React.Component {
     'count',
   ]);
 
+  sleepStartEvent = block => {
+    Events.on(block, 'sleepStart', () => {
```

Comment:
Javascript events typically bubble up so you may be able to listen on the `world` to reduce creating/destroying events for every body.

Reply:
Just realized this is internal to matterjs and not the javascript Event API so this isn't true.